### PR TITLE
Added the plugin option for wp search replace

### LINKED ISSUE #4861: Fix Broken Links in WordPress Doc Update suggest method to replace urls via plugin
Re: [Fix Broken Links in WordPress](https://pantheon.io/docs/wordpress-broken-links/)

Priority: Low

## Issue Description:
Although we have tools in doing search-replace but not all users have the technical capability or hardware to run Terminus. When users try to use the plugin method and frustrate them finding out that there is a lot of search-replace plugin that do not work out of the box.

## Suggested Resolution
Suggest a plugin that worked before with a lot of users with added screenshots. One we've seen great success is https://wordpress.org/plugins/wp-migrate-db/

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #4861

## Effect
PR includes the following changes:
- Added the plugin option for wp search replace

## Remaining Work
- [ ] List any outstanding work here
- [ ] Technical review
- [x] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Archive from **Done** in [Main Project](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/_partials/content/search-replace-domains.html [hunk 1]
```diff
@@ -4,22 +4,34 @@
...
+<li id="quicksilver-replace" role="presentation"><a href="#quicksilver-replace-anchor" aria-controls="quicksilver-replace-anchor" role="tab" data-toggle="tab">Quicksilver</a></li>  
 </ul>
 
 <!-- Tab panes -->
 
 <div class="tab-content">
 <!-- Active pane content -->
+<div role="tabpanel" class="tab-pane active" id="plugin-replace-anchor">
+
+<p>There are several plugins that does the search and replace functionality, one that reported by users that works well in the platform is [WP Migrate DB](https://wordpress.org/plugins/wp-migrate-db/). Make sure you select the search replace functionality https://screencast.com/t/Bm9woix6N</p>
```
Comment (ataylorme) [inline source/_partials/content/search-replace-domains.html]: Should the screencast link be in markdown as well? Something like 

`Make sure you select the search and replace functionality, as shown in [this screencast](https://screencast.com/t/Bm9woix6N)`

Comment (carl-alberto) [reply source/_partials/content/search-replace-domains.html]: Thanks for the feedback @ataylorme, not sure with the process of the new gatsby based docs but I was hoping @alexfornuto will be the one extracting contents and images in this PR and put it to his latest gatsby branch

Comment (alexfornuto) [reply source/_partials/content/search-replace-domains.html]: Indeed!

**Comment by alexfornuto** [comment]
Thanks!

**Pull request merged**